.test_suite.sh

```diff
@@ -26,6 +26,12 @@ testSliceMultipleRegions() {
     assertEquals 0 $?
 }
 
+testSliceMultipleRegionsBed() {
+    ./build/sambamba slice -L test/chr2_test_region.bed test/issue_204.bam |\
+      ./build/sambamba view -c /dev/stdin > /dev/null
+    assertEquals 0 $?
```

Comment:
Perhaps check at least the number of reads? (see test below, the test above is admittedly not a good example)